src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -525,7 +529,7 @@ public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
         // Add the listener for ZWave events.
         // This ensures we get called whenever there's an event we might be interested in
         if (bridgeHandler.addEventListener(this) == false) {
-            logger.error("NODE {}: Controller failed to register event handler.", nodeId);
+            logger.debug("NODE {}: Controller failed to register event handler.", nodeId);
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.